src/BuildingRegistry.Consumer.Address/Projections/CommandHandler.cs

```diff
@@ -30,5 +32,18 @@ public virtual async Task Handle<T>(T command, CancellationToken cancellationTok
 
             _logger.LogDebug($"Handled {command.GetType().FullName}");
         }
+
+        public virtual async Task HandleIdempotent<T>(T command, CancellationToken cancellationToken)
+            where T : class, IHasCommandProvenance
+        {
+            _logger.LogDebug($"Handling {command.GetType().FullName}");
+
+            await using var scope = _container.BeginLifetimeScope();
+
+            var resolver = scope.Resolve<IIdempotentCommandHandler>();
+            _ = await resolver.Dispatch(command.CreateCommandId(), command, new Dictionary<string, object>(), cancellationToken: cancellationToken);
```

Comment:
@ArneD I think it is, but just checking. Is it OK that we pass along an empty Dictionary in Dispatch?